akka-actor/src/main/scala/akka/event/ActorClassificationUnsubscriber.scala

```diff
@@ -32,17 +32,15 @@ protected[akka] class ActorClassificationUnsubscriber(bus: ManagedActorClassific
       atSeq = nextSeq
       unstashAll()
 
-    case reg: Register ⇒
-      stash()
+    case _: Register ⇒ stash()
 
     case Unregister(actor, seq) if seq == nextSeq ⇒
       if (debug) context.system.eventStream.publish(Logging.Debug(simpleName(getClass), getClass, s"unregistered watch of $actor in $bus"))
       context unwatch actor
       atSeq = nextSeq
       unstashAll()
 
-    case unreg: Unregister ⇒
-      stash()
+    case _: Unregister ⇒ stash()
```

Comment:
(I personally like the newline to be consistent with the other cases, but OK like this as well)

Reply:
I'm happy either way, reverting.

Reply:
(I still like replacing `unreg` with `_` here though ;) )